shell.nix

```diff
@@ -1,13 +1,18 @@
-with import (builtins.fetchGit rec {
-  name = "nixpkgs-19.09-${rev}";
-  url = https://github.com/nixos/nixpkgs;
-  ref = "nixos-19.09";
-  # git ls-remote https://github.com/nixos/nixpkgs-channels nixos-19.09
-  rev = "9f453eb97ffe261ff93136757cd08b522fac83b7";
-}) {};
-stdenv.mkDerivation {
+let
+  pkgs = import (builtins.fetchGit rec {
+    name = "nixpkgs-19.09-${rev}";
+    url = https://github.com/nixos/nixpkgs;
+    ref = "nixos-19.09";
+    # git ls-remote https://github.com/nixos/nixpkgs-channels nixos-19.09
+    rev = "9f453eb97ffe261ff93136757cd08b522fac83b7";
+  }) {};
+in
+pkgs.stdenv.mkDerivation {
+  passthru = {
+    inherit pkgs;
+  };
```

Comment:
bikeshed: I think this could look nicer as a one-liner, `passthru.pkgs = pkgs;`.

Or maybe `passthru = { inherit pkgs; }`?

WDYT?

Reply:
agree. done. thx :pray: